include/mbedtls/config.h

```diff
@@ -2521,7 +2521,7 @@
  *      MBEDTLS_TLS_ECDHE_PSK_WITH_ARIA_128_CBC_SHA256
  *      MBEDTLS_TLS_ECDHE_PSK_WITH_ARIA_256_CBC_SHA384
  */
-//#define MBEDTLS_ARIA_C
+#define MBEDTLS_ARIA_C
```

Comment:
I wonder, if this (and the others) are enabled by default now, and all tests assume that it is disabled, shouldn't all test cases be updated to unset these before executing tests?

Reply:
No, generally we would want to run tests with all crypto features enabled, except tests that specifically disable a crypto feature. We have such test scripts for hashes, public-key algorithms, and elliptic curves (`tests/scripts/{curves.pl,depends-*.pl}`). We don't have such a test script for ciphers; it's a known test gap. We do test with some reference configurations (`configs/*.h`) which have ARIA disabled. So there's no need for additional testing here.

Reply:
oh. alright, thanks for clarification :)